apps/site/package.json

```diff
@@ -18,7 +19,12 @@
     "sync-orama": "node ./scripts/orama-search/sync-orama-cloud.mjs",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 node --experimental-test-coverage --test-coverage-exclude=**/*.test.* --experimental-test-module-mocks --enable-source-maps --import=global-jsdom/register --import=tsx --import=tests/setup.jsx --test **/*.test.*",
     "test:unit:watch": "cross-env NODE_OPTIONS=\"--watch\" npm run test:unit",
-    "test": "turbo test:unit"
+    "test": "turbo test:unit",
+    "build-blog-data": "node ./scripts/blog-data/generate.mjs",
+    "build-blog-data:watch": "node --watch --watch-path=pages/en ./scripts/blog-data/generate.mjs",
```

Comment:
Do we need to watch the entire folder, or just the blog subdirectory?

Reply:
yeah super valid call, I think that only the blog subdirectory should we watched 🙂

Reply:
Updated, thanks 😄